build.gradle

```diff
@@ -41,9 +41,6 @@ group = 'com.cinnober.gradle'
 
 test {
   useJUnitPlatform()
-  retry {
-    maxRetries = 5
-  }
```

Comment:
Beyond the new tests not being flaky, any specific reason for removing the test retry functionality?

Reply:
It appears it was added because the tests were flaky because of a missing `proc.waitFor()`. Commit a8186d55 fixes this so the tests should be deterministic now, so the retries probably won't make any difference other than slow things down.